tracer/test/snapshots/AspNetCoreMvc30WrongMethodTests._.verified.txt

```diff
@@ -0,0 +1,28 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: POST /,
+    Service: Samples.AspNetCoreMvc30,
+    Type: web,
+    Tags: {
+      aspnet_core.endpoint: 405 HTTP Method Not Supported,
+      aspnet_core.route: ,
```

Comment:
This is an incorrect `route` tag here - we can't actually get a route. Also, I'm not sure if it's useful to have the default display name of the endpoint here, this information is already captured by the status code? Given that removing these would simplify the diagnostic observer code, probably make sense to remove them entirely?

Reply:
Fixed in 9e02c49